src/mraa.c

```diff
@@ -1233,3 +1288,668 @@ mraa_init_io(const char* desc)
     syslog(LOG_ERR, "mraa_init_io: Invalid IO type given.");
     return NULL;
 }
+
```

Comment:
The implemented functions for JSON platform could be in its own file "mraa_json.c" or something. Maybe if you think its worth it, its own directory.